src/boat/private/captain/manifest.nim.ini

```diff
@@ -0,0 +1,23 @@
+; FYI
+; 0 = false, 1 = true
```

Comment:
check if off and on will be autoparsed to bool, but i think everything is a string so dunno